src/TableContent.js

```diff
@@ -3,10 +3,40 @@ import PropTypes from 'prop-types'
 import { Table } from 'evergreen-ui'
 import URI from 'urijs'
 import ReactTooltip from 'react-tooltip'
+import _ from 'lodash'
 
 import tableHeaderCells from './tableHeaderCells'
 import TooltipIcon from './TooltipIcon'
 
+const checkInvalidFiles = async (endpoint, host, checkedRows) => {
```

Comment:
I think this function has some mixed concerns: it’s parsing a JSON payload from a non-specific endpoint (`endpoint` and `host` are variables) with a specific `{accession: checkedRows}` request parameter. I think you should either:

1. Keep the function name, remove the `endpoint` variable and push the URL building logic all the way down to this function.

2. Rename the function to e.g. `fetchJson`,  and aslo`checkedRows` to `queryParams` and pass the query object `{accession: checkedRows}` as an argument.

With option 1 you make the method more specific and with option 2 you make it more generic, but the implementation will stay true to its name.

Reply:
I refactored using option2.